src/hooks/diagnostics.ts

```diff
@@ -10,25 +10,22 @@ import { NpmModule } from '../shared/npmCommand.js';
 type HookFunction = (options: { doctor: SfDoctor }) => Promise<[void]>;
 export const hook: HookFunction = (options) => Promise.all([registryCheck(options)]);
 
-// eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
require-await and floating-promises below are valid errors.  that map>async thing shouldn't return until the async calls are done running.